### ISSUE TITLE
Feat: Use real HTML links for dashboard buttons

### ISSUE THREAD

**Issue by raphj** [tracim/tracim#3807]
They are currently divs and cannot be used without a mouse.

Links would be better than buttons since they use routes to send the user to another page.

**Comment by gbonaspetti** [comment]
Fixed at #4353
